Custom route for backstage backend for backstage-plugin-opa-authz-react and manually triggerable hook (#281)
Parsifal-M/backstage-opa-plugins: plugins/backstage-plugin-opa-authz-react/README.md

};
```

### Alternative to using the `useOpaAuthz` you can use `useOpaAuthzManual` hook (also optional)

> If you need more control over when the policy evaluation is triggered, you can use the `useOpaAuthzManual` hook. This hook provides a `triggerFetch` function that allows you to manually trigger the policy evaluation:

Reviewer: ```suggestion
If you need more control over when the policy evaluation is triggered, you can use the `useOpaAuthzManual` hook. This hook provides an `evaluatePolicy` function that allows you to manually trigger the policy evaluation:
```

```tsx
import React, { useEffect } from 'react';
import { useOpaAuthzManual } from '@parsifal-m/backstage-plugin-opa-authz-react';

const MyComponent = () => {
  const [userData, setUserData] = React.useState(null);

>   const { loading, data, error, triggerFetch } = useOpaAuthzManual(

Reviewer: ```suggestion
  const { loading, data, error, evaluatePolicy } = useOpaAuthzManual(
```

    { action: 'read-policy' },
    'authz',
  );

  useEffect(() => {
    const fetchData = async () => {
      // First fetch some user data
      const response = await fetch('/api/user-data');
      const userData = await response.json();
      setUserData(userData);

      // Then evaluate the policy with the fetched data

>       const result = await triggerFetch();

Reviewer: ```suggestion
      const result = await evaluatePolicy();
```

      if (result?.result.allow) {
        // Do something when access is granted
      }
    };

    fetchData();

>   }, [triggerFetch]);

Reviewer: ```suggestion
  }, [evaluatePolicy]);
```
